docs/source/guide/developer/1-modulebase.md

```diff
@@ -21,7 +21,7 @@ This process requires four basic class in EvoX:
 It is necessary to provide a unified module for them. In EvoX, the four classes are all inherited from the base module — [`ModuleBase`](#evox.core.module.ModuleBase).
 
 <center>
```

Comment:
You don't need html code for this, and the path is `/_static/modulebase.png`
~~~
```{image} /_static/modulebase.png
:alt: Module base
:align: center
```
~~~